app/src/main/java/com/github/leuludyha/ibdb/presentation/components/utils/WithHeader.kt

```diff
@@ -0,0 +1,23 @@
+package com.github.leuludyha.ibdb.presentation.components.utils
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun WithHeader(
```

Comment:
Nitpick: I know it is a very simple component, but I still think that some documentation would be great, since its use is meant to be very generalized.